codebase/codebase.go

```diff
@@ -0,0 +1,61 @@
+package codebase
+
+import "errors"
+
+// Codebase defines all parameters those are useful to associate Che Editor's window to a WI
+type Codebase struct {
```

Comment:
Thanks @aslakknutsen , now renamed to `CodebaseContent` in https://github.com/almighty/almighty-core/pull/907/commits/4c390be67ed04c4d8a8f1925cb9cdb84bb5d100f